src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Performance/DoNotUseCountWhenAnyCanBeUsedTests.Tests.cs

```diff
@@ -458,6 +458,7 @@ End Function
     Sub M()
         Dim b As Boolean
 
+        b = {{|{this.Verifier.DiagnosticId}:GetData().Count.Equals(IntegerZero)|}}
```

Comment:
Can you add this non parentheses scenario for `0.Equals(GetData().Count)`, `GetData().Count = 0` and  `0 = GetData().Count` cases below?